### DISCUSSION TITLE
NoSecurityIsolation for groups

### DISCUSSION THREAD

**Discussion by MbugR** [sandboxie-plus/Sandboxie#3613] [Q&A Feedback]
Hello, can i use NoSecurityIsolation for groups and without NoSecurityIsolation for other groups?

**Comment by offhub** [comment]
Groups are visual only. You can use the `NoSecurityIsolation` setting as a box-specific or global setting.

**Comment by MbugR** [comment]
I need to use NoSecurityIsolation for certain processes for compatibility
